eng/pipelines/coreclr/perf-non-wasm-jobs.yml

```diff
@@ -2,24 +2,159 @@ jobs:
 
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'Schedule')) }}:
 
-  # build mono
+  # build mono iOS scenarios HybridGlobalization
```

Comment:
If all these new jobs are just copies of the existing jobs for ios/scenarios, then would it make sense to make the whole of ios/scenarios a template with `hybridGlobalization` as a parameter?

Reply:
This is something I will look into outside this PR, there is probably some general level cleanup and consolidation that can be done to make the yml more manageable.